src/Players.java

```diff
@@ -0,0 +1,27 @@
+public class Players {
+
+    int points = 0;
+
+    public Players() {
+
+    }
+
+    public void add () {
+        points = points +1;
+    }
+
+    public void sub () {
+        points = points - 1;
```

Comment:
Nitpick: you should probably don't allow negative numbers

Reply:
oh...you're right! But the __sub method__ will be changed into a __set method__ in the next step